api/common/conf.py

```diff
@@ -38,14 +38,9 @@ def __init__(self):
         self.LOCAL_DATA_PATH = os.environ.get("LOCAL_STATIC_DATA_PATH", None)
 
         self.RECALL_PETITION_ENABLED = True
-
-        self.PARL_BOUNDARY_CHANGES_ENABLED = True
         self.PARL_BOUNDARY_DATA_KEY_PREFIX = (
             "parl-2023-boundary-review/20240603"
         )
```

Comment:
I'm leaving this here because it is used in the `StaticDataHelper` base class
https://github.com/DemocracyClub/aggregator-api/blob/0d9c876c619979392211d41a4c346a3a3213b80c/api/endpoints/v1/voting_information/static_data_helper.py#L27
which is, in turn, used by the `RecallPetitionApiClient`.

Tbh, this smells fishy to me though. I wonder if that is actually right?

Reply:
Yeah, I think the method in the helper should be abstract. It's not used in `RecallPetitionApiClient` as that class implements `get_file_path`.

Reply:
Right yeah I didn't think to check this. Both classes that inherit from this implement it. In https://github.com/DemocracyClub/aggregator-api/pull/580/commits/7beaaf876986454b8d4df2f209d13b62e1d84e84 I have converted this to abstract and deleted the setting.